--- aiohttp-3.6.3/setup.py~	2020-10-12 10:52:44.000000000 +0000
+++ aiohttp-3.6.3/setup.py	2020-10-12 17:28:42.936849018 +0000
@@ -74,9 +74,9 @@
 install_requires = [
     'attrs>=17.3.0',
     'chardet>=2.0,<4.0',
-    'multidict>=4.5,<5.0',
+    'multidict>=4.5',
     'async_timeout>=3.0,<4.0',
-    'yarl>=1.0,<1.6.0',
+    'yarl>=1.0',
     'idna-ssl>=1.0; python_version<"3.7"',
     'typing_extensions>=3.6.5; python_version<"3.7"',
 ]
--- aiohttp-3.6.3/aiohttp.egg-info/requires.txt~	2020-10-12 10:52:55.000000000 +0000
+++ aiohttp-3.6.3/aiohttp.egg-info/requires.txt	2020-10-12 17:29:06.795850902 +0000
@@ -1,8 +1,8 @@
 attrs>=17.3.0
 chardet<4.0,>=2.0
-multidict<5.0,>=4.5
+multidict>=4.5
 async_timeout<4.0,>=3.0
-yarl<1.6.0,>=1.0
+yarl>=1.0
 
 [:python_version < "3.7"]
 idna-ssl>=1.0
